src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,13 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+	if (number % 3 === 0 && number % 5 === 0) {
+		return "FizzBuzz";
+	} else if (number % 5 === 0) {
+		return "Buzz";
+	} else if (number % 3 === 0) {
+		return "Fizz";
+	} else {
+		return number;
+	}
```

Comment:
This is some pretty extreme indentation you have here! Normally the convention is to use either 2 or 4 spaces throughout, it looks here like youre using tabs instead, is that right?

Either way, this works, but since youre using the same logic a few times over, you could put this into variables to make the code a bit more readable, eg:
```
  const fizz = number % 3 === 0;
  const buzz = number % 5 === 0;
  if (fizz && buzz) {
    return "FizzBuzz";
  }
  if (buzz) {
    return "Buzz";
  }
  if (fizz) {
    return "Fizz";
  }
  return number;
```